pkg/sql/sem/plpgsqltree/statements.go

```diff
@@ -115,6 +127,30 @@ func (s *PLpgSQLStmtIf) Format(ctx *tree.FmtCtx) {
 	ctx.WriteString("<NOT DONE YET>")
 }
 
+func (s *PLpgSQLStmtIf) walkStmt(visitor plpgsqlVisitor) {
```

Comment:
Same, we should use interface type in the signature. same comment on others as well.